### ISSUE TITLE
Add 6 months and yearly shortcuts with week / month intervals

### ISSUE THREAD

**Issue by adriaanvanrossum** [simpleanalytics/roadmap#11]
The URLs have now start and end dates (/example.com?start=2018-09-14&end=2019-07-27) but it would be nice to have shortcuts like this (/example.com?start=yesterday&end=today or /example.com?period=month or /example.com?period=6-months).

Thanks to @JustinNoelDev on Twitter

**Comment by calendee** [comment]
Thanks so much for implementing the date ranges!  For longer ranges (6 months?), it would be great if the stats could be grouped by week.  For example, this is a year of my site:

<img width="433" alt="simple-analytics-annual" src="https://user-images.githubusercontent.com/57374/70487640-9b9a2b00-1abb-11ea-9211-289d113f0c31.png">

It's not very readable nor easy to see a real trend since every single day is plotted.  If it was plotted by the total per week, it would present more useful information.

**Comment by adriaanvanrossum** [comment]
You're totally right! Thanks. We are updating our database to handle queries easily for this. But we first our moving our servers to The Netherlands, so it could take a while before it's implemented. But we are definitely going to do this!

**Comment by adriaanvanrossum** [comment]
Another customer shared this use case:

> I often like to view my site usage statistics for today's date. I do it often enough that it'd be nice to create a bookmark in my browser that goes direct to the page for today's stats.
> 
> However, the URL for that page is something like:
> 
> https://simpleanalytics.com/example.com?start=2020-05-20&end=2020-05-20
> 
> And since it has today's date 2020-05-20 in the URL, this will be out of date for me when tomorrow rolls around.
> 
> Any chance you could add a shortcut like this instead:
> 
> https://simpleanalytics.com/example.com?start=today
> 
> And this link would automatically redirect me to the page for today's date? That way, I could bookmark the shortcut and save a bunch of time...

That makes a lot of sense. Lets do this one in the near future @hellodeibu!

I would suggest we use `period=today` together with `start=yesterday&end=today` like helpers. Maybe shortcuts like `-3days` and `-5months` would be cool as well.

**Comment by adriaanvanrossum** [comment]
This will be part of the beta

**Comment by adriaanvanrossum** [comment]
@hellodeibu Could you make this work like @calendee describes?
